sagemaker-keyspaces/SageMaker_Keyspaces_ml_example.ipynb

```diff
@@ -0,0 +1,469 @@
+{
```

Comment:
<p>I think it is sufficient to provide the IAM policy here, rather than in the standalone txt file.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Thank you.  I will remove the file </p>